config/onprem-iso-config.ign

```diff
@@ -56,7 +56,7 @@
         "name": "assisted-service-db.service"
       },
       {
-        "contents": "[Unit]\nAfter=assisted-service-db.service\n\n[Service]\nType=forking\nRestart=no\nExecStartPre=podman pull quay.io/rwsu/assisted-service:test\nExecStart=podman run -dt --pod assisted-installer --env-file /etc/assisted-service/environment --restart always --name installer quay.io/rwsu/assisted-service:test\nTimeoutStartSec=300\n\n[Install]\nWantedBy=multi-user.target\n",
+        "contents": "[Unit]\nAfter=assisted-service-db.service\n\n[Service]\nType=forking\nRestart=no\nExecStartPre=podman pull quay.io/rwsu/assisted-service:dns\nExecStart=podman run -dt --pod assisted-installer --env-file /etc/assisted-service/environment --restart always --name installer quay.io/rwsu/assisted-service:dns\nTimeoutStartSec=300\n\n[Install]\nWantedBy=multi-user.target\n",
```

Comment:
Should be able to override the image, and default to quay.io/ocpmetal/assisted-service:latest

Reply:
I've changed the default to quay.io/ocpmetal/assisted-service:latest and added additional notes in the doc on the need to use a custom assisted-service container image for now. quay.io/ocpmetal/assisted-service:latest doesn't contain the extra bits that are needed for onprem. We can remove the need for a custom container image when we have a single container image for both cloud-based and onprem, and more specifically when Dockerfile.assisted-service-onprem is merged into Dockerfile.assisted-service.